build.gradle.kts

```diff
@@ -45,7 +47,7 @@ dependencies {
   testImplementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
   testImplementation("com.github.tomakehurst:wiremock-jre8-standalone:2.35.0")
   testImplementation("org.mockito:mockito-inline:5.2.0")
-  testImplementation("io.opentelemetry:opentelemetry-sdk-testing:1.25.0")
+  testImplementation("io.opentelemetry:opentelemetry-sdk-testing:1.26.0")
```

Comment:
this should be linked to the app insights upgrade - might be better to go to 5.2.0-beta to grab that as well